### PR TITLE
Add Google Cloud Storage step

### PR DESCRIPTION
**Pull request by remi**

### CODE REVIEW

File: aspnet/3-binary-data/Scripts/jquery.validate.js [hunk 1]
```diff
@@ -0,0 +1,1398 @@
+/*!
```
Comment (SurferJeffAtGoogle) [inline aspnet/3-binary-data/Scripts/jquery.validate.js]: Drop all the files in Scripts/...

File: aspnet/3-binary-data/Content/bootstrap-theme.css [hunk 2]
```diff
@@ -0,0 +1,587 @@
+/*!
```
Comment (SurferJeffAtGoogle) [inline aspnet/3-binary-data/Content/bootstrap-theme.css]: Drop all the files in Content/...

Comment (remi) [inline aspnet/3-binary-data/Content/bootstrap-theme.css]: Hmmm ... I wonder how these got in here.  I copy/pasted 2-structured-data.  I should test to see if VS adds these if they're missing, eg. for caching (unless you already know)

I'll remove these and add the path to .gitignore

**Comment by SurferJeffAtGoogle** [comment]
After this PR, please update runTests.ps1 and applicationhost.config.

**Pull request merged**